EIPS/eip-5187.md

```diff
@@ -0,0 +1,102 @@
+
+## Motivation
+The traditional [EIP-721](./eip-721.md) and EIP-1155 focus more on ownership. However, NFTs as digital assets are more prominent in use than ownership. Taking artistic NFTs as an example, NFT artists may wish to rent out the use rights of their works to media companies for an allotted time, or NFT musicians may wish to make their music available to listeners as per playing duration. 
+Therefore, to better serve NFT developers to meet such needs and develop more sophisticated NFT products, we propose directly introducing rentable usage rights to complement the ERC standard.
+
+
+## Specification
+The key words “MUST”, “MUST NOT”, “REQUIRED”, “SHALL”, “SHALL NOT”, “SHOULD”, “SHOULD NOT”, “RECOMMENDED”, “MAY”, and “OPTIONAL” in this document are to be interpreted as described in RFC 2119.
+
+```solidity
+pragma solidity ^0.8.0;
+
+interface IRental is IERC1155 {
+    /**
+     * @notice This emits when user rent NFT
+     * @param id The id of the current token
+     * @param user The address to rent the NFT usage rights
+     * @param amount The amount of usage rights
+     * @param expires The specified period of time to rent
+     **/
+    event Rented(
+        uint256 indexed id,
+        address indexed user,
+        uint256 amount,
+        uint256 expires
+    );
+
+    /**
+     * @notice This emits when the NFT owner takes back the usage rights from the tenant (the `user`) 
+     * @param id The id of the current token
+     * @param user The address to rent the NFT's usage rights
+     * @param amount Amount of usage rights
+     **/
+    event TakeBack(uint256 indexed id, address indexed user, uint256 amount);
+
+    /**
+     * @notice Function to rent out usage rights
+     * @param from The address to approve
+     * @param to The address to rent the NFT usage rights
+     * @param id The id of the current token
+     * @param amount The amount of usage rights
+     * @param expires The specified period of time to rent
+     **/
+    function safeRent(
+        address from,
+        address to,
+        uint256 id,
+        uint256 amount,
+        uint256 expires
+    ) external;
+
+    /**
+     * @notice Function to take back usage rights after the end of the tenancy
+     * @param user The address to rent the NFT's usage rights
+     * @param tokenId The id of the current token
+     * @param amount The amount of usage rights
+     **/
+    function takeBack(
+        address user,
+        uint256 tokenId,
+        uint256 amount
+    ) external;
+
+    /**
+    * @notice Return the NFT to the address of the NFT property right owner.
+    **/
+    function propertyRightOf(uint256 id) external view returns (address);
+
+    /**
+    * @notice Return the total supply amount of the current token
+    **/
+    function totalSupply(uint256 id) external view returns (uint256);
```

Comment:
This is in the original 1155. Unless you are specifying a different behavior, which I'd suggest an explicit call out, otherwise, suggest removing it.

Reply:
Actually, I'm pretty sure it isn't. I believe it *is* in the `IERC1155Supply` extension.